### DISCUSSION TITLE
Vorlage / Template für Geräte gleichen Typs?

### DISCUSSION THREAD

**Discussion by RealKodijack** [Zefau/ioBroker.jarvis#1878] [Q&A]
Hi!
Ich habe zahlreiche Änderungn an Datentypen in zahlreichen Geräten. 
Jedoch fand ich nach langer Suche keine Möglichkeit eine Vorlage zu nutzen statt jedes einzelne Gerät per copy&paste zu ändern. Mit steigender Anzahl an Geräten steigt das Chaos.

Falls nicht klar ist was ich meine:

DisplayVariant in <Vorlage: function=>light> = <blabla> anstatt DisplayVariant <blabla> in jedem einzelnen Gerät.

Wär super, wenn jemand eine Idee hätte.

**Comment by Zefau** [comment]
via https://github.com/Zefau/ioBroker.jarvis/issues/188
